installer/create_installer.sh

```diff
@@ -19,11 +19,6 @@ function git_show {
 
 cd "$(dirname "$0")"
 
-echo -e "${BYELLOW}This script must be run from the installer directory!${RESET}"
-echo "The current working directory is $(pwd)"
-read -p "If that looks right, press any key to proceed, or CTRL-C to exit..."
-echo
-
```

Comment:
I removed this because it was fiddly to make work with the workflows. The previous command ensures the directory is set correctly anyways.